lib/twitter/streaming/client.rb

```diff
@@ -132,7 +132,7 @@ def request(method, uri, params)
 
       def to_url_params(params)
         params.collect do |param, value|
-          [param, URI.encode(value)].join('=')
+          [param, CGI.escape(value)].join('=')
```

Comment:
Should we add `require 'cgi'` to this file for completeness? It looks like it's required in a few others, and our Rails envs will have it loaded by other things, but maybe it's best to be explicit?